molecule/docker/molecule.yml

```diff
@@ -0,0 +1,33 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: docker
+lint:
+  name: yamllint
+platforms:
+  - name: docker
+    image: replicajune/ubuntu-dind-supervisord
+    docker_host: "${DOCKER_HOST:-unix://var/run/docker.sock}"
+    privileged: true
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+      - /var/run/docker.sock:/var/run/docker.sock:rw
+      - /root/.ansible:/root/.ansible:rw
+      - /home:/home:rw
```

Comment:
Why `/home` is mounted as read-write?

Reply:
That is because I couldn't predict the user use to run ansible, is it molecule ?